### ISSUE TITLE
Can't get beta 1.7.3 far lands to generate above y=128

### ISSUE THREAD

**Issue by xp2-882030kgz010602** [Nostalgica-Reverie/moderner-beta#20]
I have tried changing the "height" tag (line 15) from 128 to 256 in both beta.json and modern_beta.json, but I cannot get the far lands to generate above y=128. I know that it should be possible, since those same far lands generate below y=0 into deepslate, while the "real" beta 1.7.3 couldn't have done this at all.

**Comment by xp2-882030kgz010602** [comment]
Update: I've found that there's a `worldHeight` parameter in `ModernBetaChunkGenerator`, but I can't figure out how that gets initialized since it seems to go back into MC code.

**Comment by BlueStaggo** [comment]
Yeah, I should make the maximum height configurable in the chunk provider instead of being dependent on the noise config.

**Comment by New-Account-5728415** [comment]
How difficult would this be to add in? Would there need to be a way to patch already generated worlds with a new ?
